airflow/providers/ftp/hooks/ftp.py

```diff
@@ -88,9 +90,10 @@ def list_directory(self, path: str) -> list[str]:
         :param path: full path to the remote directory to list
         """
         conn = self.get_conn()
+        current_path = conn.pwd()
         conn.cwd(path)
-
         files = conn.nlst()
+        conn.cwd(current_path)
```

Comment:
same here (https://pythontic.com/ftplib/ftp/nlst):
```suggestion
        files = conn.nlst(path=path)
```

Reply:
Thank a lot. It's look better. Cant commit your suggestion because of typo: `nlst` method has no `path` name argument

https://github.com/apache/airflow/pull/35105/commits/eb1c5cb22c4ba1bd023036b99733f1e4a75be289

Reply:
@hussein-awala can you see the changes you requested?